docs/qupath.rst

```diff
@@ -1,69 +1,115 @@
 
   .. image:: images/qupath1.png
 
-#.  Click *OK*. In the following dialog, enter your credentials.
+#. Click *OK*. In the following dialog, enter your credentials.
 
   .. image:: images/qupath2.png
 
-#.  Set image type to *Brightfield H&E* in the following dialog.
+#. Set image type to *Brightfield H&E* in the following dialog.
 
-#.  Find a region with well-defined cells and nuclei in the image, zoom
-    in.
+#. Find a region with well-defined cells and nuclei in the image, zoom in.
 
-#. Draw an ROI Annotation which denotes the region in which the cells
-   will be detected using the *Wand* tool 
+#. Draw an ROI Annotation which denotes the region in which the cells will be detected using the *Wand* tool. 
 
   .. image:: images/qupath3.png
 
 #. Adjust your ROI Annotation using the *Brush* tool.
 
   .. image:: images/qupath4.png
 
-#. Select *Analyze > Cell analysis > Cell* detection
+#. Select *Analyze > Cell analysis > Cell* detection.
 
-#. You can adjust the parameters. Click *Run*. This will draw red ROIs
-    around cells and nuclei inside your ROI Annotation.
+#. You can adjust the parameters. Click *Run*. This will draw red ROIs around cells and nuclei inside your ROI Annotation.
 
   .. image:: images/qupath5.png
 
-#. Select *Measure > Show detection measurements*
+#. Select *Measure > Show detection measurements*.
 
   .. image:: images/qupath6.png
 
-#. Note: You can save the results locally by clicking *Save* in the
-    bottom right of the *Detection results table*.
+#. Note: You can save the results locally by clicking *Save* in the bottom right of the *Detection results table*.
 
-#. Select *Automate > Create* command history script.
+#. In the following steps, we will show how to convert the ROIs your just created in QuPath into OMERO ROIs and attach them to the image in OMERO.
 
-#. This will automatically create a new script from the sequence of
-    commands you just run and open it in the *Script editor*.
+#. First, use the ROI OME-XML export script to export your ROIs from QuPath into OME-XML file. See https://github.com/glencoesoftware/ome-omero-roitool/blob/master/QuPath.scripts/OME_XML_export.groovy#L23
 
-#. Note: This script does not capture the opening of images from OMERO.
-    Also, we want to add parts where the ROIs from our *Detection* are
-    saved on the image in OMERO. That is why we adjusted the script
-    by adding the missing parts (opening of images from OMERO and
-    saving the ROIs and result tables back to OMERO) and created a
-    new script which we will run here.
+#. Note: The version of the ROI OME-XML export script you have to use depends on the version of QuPath you are using. For QuPath 0.2.0-m9 or earlier versions, use https://github.com/glencoesoftware/ome-omero-roitool/blob/master/QuPath.scripts/OME_XML_export.groovy. For QuPath 0.2.0-m10 use https://github.com/glencoesoftware/ome-omero-roitool/blob/d73f14a68e35eb1f9d1cc810255a42e4fa46d283/QuPath.scripts/OME_XML_export.groovy.
```

Comment:
Since it is internal milestone. This could be removed when an official version is out